lazer/contracts/aptos/sources/pyth_lazer.move

```diff
@@ -0,0 +1,132 @@
+module pyth_lazer::pyth_lazer {
+    use std::vector;
+    use std::signer;
+    use aptos_framework::timestamp;
+    use aptos_framework::coin;
+    use aptos_framework::aptos_coin::AptosCoin;
+    use aptos_std::ed25519;
+
+    /// Error codes
+    const ENO_PERMISSIONS: u64 = 1;
+    const EINVALID_SIGNER: u64 = 2;
+    const ENO_SPACE: u64 = 3;
+    const ENO_SUCH_PUBKEY: u64 = 4;
+    const EINVALID_SIGNATURE: u64 = 5;
+    const EINSUFFICIENT_FEE: u64 = 6;
+
+    /// Constants
+    const MAX_NUM_TRUSTED_SIGNERS: u8 = 2;
```

Comment:
let's actually remove this MAX. I don't think we should've had it in the solana contract either.